src/lib/features/feature-toggle/fake-feature-strategies-read-model.ts

```diff
@@ -0,0 +1,20 @@
+import type { IFeatureStrategiesReadModel } from './types/feature-strategies-read-model-type';
+
+export class FakeFeatureStrategiesReadModel
+    implements IFeatureStrategiesReadModel
```

Comment:
Just wondering, would it be better to call this a `FeatureStrategiesMetrictsReadModel` or someting? Not a hard suggestion, just thought it would be clearer. This read model could be mistaken for something that returns flag strats 💁🏼